graphql/schema/types/config.graphql

```diff
@@ -17,6 +17,11 @@ type SpruceConfig {
   ui: UIConfig!
 }
 
+type AdminSettings {
+  banner: String
+  bannerTheme: String
```

Comment:
I wonder if it possible to use our GraphQL enum that already exists, `BannerTheme`? It'd be nice because then the values would be strongly typed (since it's not technically any string)
```suggestion
  bannerTheme: BannerTheme
```